<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>97</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Shopping Cost</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>53</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculateCost">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate Cost</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>80</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="breadCheckBox">
      <property name="text">
       <string>Bread</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="potatoCheckBox">
      <property name="text">
       <string>Potato</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="milkCheckBox">
      <property name="text">
       <string>Milk</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="eggsCheckBox">
      <property name="text">
       <string>Eggs</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="butterCheckBox">
      <property name="text">
       <string>Butter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="tomatoCheckBox">
      <property name="text">
       <string>Tomato</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>89</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Per Cost: 0.8$</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Per Kilo: 1$</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Per Kilo: 3$</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Per Cost: 0.1$</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Per Kilo: 13$</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Per Kilo: 1.5$</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>71</y>
     <width>156</width>
     <height>193</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLineEdit" name="breadAmount">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="potatoAmount">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="milkAmount">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="eggsAmount">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="butterAmount">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="tomatoAmount">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>breadCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>breadAmount</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>potatoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>potatoAmount</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>milkCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>milkAmount</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eggsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>eggsAmount</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>butterCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>butterAmount</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tomatoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>tomatoAmount</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
